### PR TITLE
Add a delete button to the global configurations.

### PR DESCRIPTION
**Pull request by psantos1113**
Added a delete button with a confirmation modal to the config buttons.  The global configurations, provider and channel, will display a delete button to delete the configurations.

### CODE REVIEW

File: src/main/js/component/common/ConfirmModal.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                    <Modal.Title>{title}</Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    {message}
+                </Modal.Body>
+                <Modal.Footer>
+                    <button id="confirmNegative" type="button" className="btn btn-link" onClick={this.handleNegativeClick}>{negativeButtonText}</button>
+                    <button id="confirmAffirmative" type="button" className="btn btn-danger" onClick={this.handleAffirmativeClick}>{affirmativeButtonText}</button>
+                </Modal.Footer>
+            </Modal>
```
Comment (bamandel) [inline src/main/js/component/common/ConfirmModal.js]: Do we have a modal component that can be used effectively here?

Comment (gkillough) [reply src/main/js/component/common/ConfirmModal.js]: I think this component is exactly that.

Comment (psantos1113) [reply src/main/js/component/common/ConfirmModal.js]: You're talking about Popup.js which should move into the component folder.  But that uses our config buttons.  This is to confirm an action and it has the same look and feel as LogoutConfirmation. Which is also a confirmation dialog.

Comment (bamandel) [reply src/main/js/component/common/ConfirmModal.js]: We could try building on top of the pop up button a couple different types of pop ups. But that would be for the next version

**Pull request merged**